test/integration/external/fire_cloud_client_test.rb

```diff
@@ -270,22 +270,22 @@ def test_get_submission_queue_status
   end
 
   # get available workflows
-  def test_get_methods
-    skip if @smoke_test
-    # get all available methods
-    workflow_methods = @fire_cloud_client.get_methods(entityType: 'workflow')
-    assert workflow_methods.any?, 'Did not find any workflow methods'
-
-    # get a single method
-    method_params = workflow_methods.sample
-    method = @fire_cloud_client.get_method(method_params['namespace'], method_params['name'], method_params['snapshotId'])
-    assert method.present?, "Did not retrieve requested method: '#{method_params['namespace']}/#{method_params['name']}/#{method_params['snapshotId']}'"
-
-    # get a method payload
-    method_payload = @fire_cloud_client.get_method(method_params['namespace'], method_params['name'], method_params['snapshotId'], true)
-    assert method_payload.present?, "Did not retrieve requested method payload: '#{method_params['namespace']}/#{method_params['name']}/#{method_params['snapshotId']}'"
-    assert method_payload.is_a?(String), "Method payload is wrong type, expected String but found #{method_payload.class}"
-  end
+  # def test_get_methods
```

Comment:
For maintenance context:

```suggestion
  # These tests are disabled because they sometimes throw false positives, and SCP hasn't support workflows for a long time
  # def test_get_methods
```

Reply:
[a8deffe](https://github.com/broadinstitute/single_cell_portal_core/pull/2189/commits/a8deffe4a1b7aa770758e8c168abe19fdb6c9a3b)